kubernetes-client/src/main/java/io/fabric8/kubernetes/client/dsl/internal/KubernetesListOperationsImpl.java

```diff
@@ -126,7 +126,7 @@ public KubernetesList get() {
   }
 
   private <T extends HasMetadata, V extends VisitableBuilder<T, V>> T create(T resource) {
-    ResourceHandler<T, V> handler = Handlers.get(resource.getKind(), resource.getApiVersion());
+    ResourceHandler<T, ?> handler = Handlers.get(resource);
```

Comment:
Why not `ResourceHAndler<T, V> handler`?

If not possible, then the V type parameter declaration should be dropped from the signature

Reply:
We can use V there now.  Initially Handlers was more loosely typed.

Reply:
Updated now.